infra/helper.py

```diff
@@ -873,6 +876,11 @@ def shell(args):
       '%s:/work' % _get_work_dir(args.project_name), '-t',
       'gcr.io/%s/%s' % (image_project, args.project_name), '/bin/bash'
   ]
+  if args.source_path:
```

Comment:
may I ask you to change the lines above to be:

```python
run_args = _env_to_docker_args(env)
if args.source_path:
  run_args.extend(['-v', '%s:%s' % (_get_absolute_path(args.source_path), '/src')])
run_args.extend([
    # <lines 874-877 here>
])
```

Reply:
Sure. Sorry for a delay. Should be fixed in current version.